recipes/openblas/bld.bat

```diff
@@ -0,0 +1,11 @@
+
+:: Set $HOME to the current dir so msys runs here
+set HOME=%cd%
+
+:: Configure, build, test, and install using `make`.
+bash -lc "make FC=gfortran DYNAMIC_ARCH=1 BINARY=$ARCH NO_LAPACK=0 NO_AFFINITY=1 USE_THREAD=1 PREFIX=`echo $LIBRARY_PREFIX | tr '\\' '/'`"
```

Comment:
No worries :-) I find windows to be a vast and alien architecture myself, I've just been forced to spend more time exploring its non-Euclidean crannies.

Note also that because openblas needs inline asm and Fortran, you can't really compile it with msvc anyway. (Technically it supports it now I think, but you lose LAPACK and all optimized kernels so it's pretty useless.)